src/components/WebcamCapture/WebcamCapture.test.js

```diff
@@ -3,17 +3,10 @@ import {shallow, configure} from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import WebcamCapture from './WebcamCapture';
 import Webcam from 'react-webcam';
-import Notification from './../Notification/Notification';
 
 configure({adapter: new Adapter()});
 
 describe('<WebcamCapture />', () => {
```

Comment:
do you want to write a test?

Reply:
Finding it a bit tricky to get it to fail in test without changing the code